modules/katello_content_view.py

```diff
@@ -164,7 +162,8 @@ def main():
     )
 
     if not HAS_NAILGUN_PACKAGE:
-        module.fail_json(msg="Missing required nailgun module (check docs or install with: pip install nailgun")
+        module.fail_json(msg="Missing required nailgun module (check docs or install with: "
+                         "pip install nailgun")
```

Comment:
I'd argue we should follow whatever Pep8/Ansible standards are since our ultimate goal is putting these into Ansible proper.